src/trove/lists.scrbl

```diff
@@ -25,30 +25,6 @@
 '(module
   "lists"
   (path "src/arr/base/lists.arr")
-  (unknown-item
```

Comment:
These bindings are present in the lists file, at least for now, so technically they should stay here.  The call to `@ignore` below lets us know in this documentation file that we don't intend to explain them.  It's a holdover from when we autogenerated the skeletons of "which functions do I need to document for this file?"

Reply:
Not anymore :) So I won't do anything regarding this.